text/0118-mobile-transactions-and-spans.md

```diff
@@ -65,3 +75,18 @@ Have one transaction per screen. This automatic transaction can be fully managed
 ### 3. Leave it as-is
 
 Whilst being the least effort, this option doesn't add any value and we remain with all the drawbacks as outlined in the [background section](#background).
+
+### 4. Single Span Ingestion <a name="option-4"></a>
```

Comment:
sidenote/question: why is it "single" span ingestion? I know we didn't invent this here, maybe you have a source

Reply:
That's just the wording people always brought up. I don't have any source.